docs/advanced_topics/customisation/rich_text_internals.rst

```diff
@@ -97,6 +97,23 @@ The ``register_link_type`` method allows you to define a function to be called w
     def register_report_link(features):
         features.register_link_type('report', report_link_handler)
 
+It is also possible to define link rewrite handler for the External Links (``external``) and Email Links (``email``). For example, if you want to open your external links in new window, You can write a custom link rewrite handler for ``external`` links.
```

Comment:
I'll change this to be about `rel="nofollow"`. Although `target="_blank"` is the use case we've got the most requests for, it's questionable whether implementing it is a good idea (see conversation in #1167). Ideally, Wagtail docs should use examples that are uncontroversial so they are more likely to stay relevant over time.